versions/development/parsers/grammar.hgr

```diff
@@ -108,6 +108,7 @@ grammar {
 
     mode<awaiting_version_name> {
       r'1.1-draft' -> :version_name %pop
+      r'biscayne' -> :version_name %pop
```

Comment:
I don't think that `biscayne` should be in this at all, unless you'd like to propose that as the official name for `not-yet-1.1`

Reply:
I'm happier with 'biscayne' than 1.1-draft. biscayne forever in /dev, real numbers in /!dev

Reply:
I'm happy to advocate for something other than 1.1 (since who knows what the version number will be if any of the breaking changes make it in)

Reply:
It's more that at the moment `biscayne` is meaningless in WDL-world

I agree that `1.1` is not the right value.

Reply:
I dont really know what biscayne is.. afaik it's a cromwell thing. I do not think it belongs in this pr. I would probably move to a patch version in this instance rather then introduce some weird code name that makes  no sense